lib/vmdb/loggers.rb

```diff
@@ -132,11 +132,9 @@ def self.create_loggers
     private_class_method :create_loggers
 
     def self.create_multicast_logger(log_file_path, logger_class = VMDBLogger)
-      logger_instance = logger_class.new(log_file_path).tap do |logger|
+      logger_class.new(log_file_path).tap do |logger|
         logger.level = Logger::DEBUG
```

Comment:
Do we still need this settings of DEBUG line?  If we create the logger and leave it with the default, it should Just Work™

Reply:
Probably don't need this now.  I can remove.